cmake/modules/FindFFMPEG.cmake

```diff
@@ -166,6 +166,17 @@ if(NOT ENABLE_INTERNAL_FFMPEG OR KODI_DEPENDSBUILD)
                                                     FFMPEG_VERSION
                                       FAIL_MESSAGE "FFmpeg ${REQUIRED_FFMPEG_VERSION} not found, please consider using -DENABLE_INTERNAL_FFMPEG=ON")
 
+    include(CheckCSourceCompiles)
+    set(CMAKE_REQUIRED_INCLUDES ${FFMPEG_INCLUDE_DIRS})
+    check_c_source_compiles("#include <libavutil/hwcontext_drm.h>
+
+                             int main()
+                             {
+                               AVDRMFrameDescriptor test;
+                               return test.format;
+                             }
+                             " FFMPEG_HAS_AVDRMFRAMEDESCRIPTOR_FORMAT)
```

Comment:
For the FFmpeg version Kodi requires, this check will always be false, so there is no need for it.

Reply:
Correct, this is depending on a feature not yet part of ffmpeg, see [drmprime-format](https://github.com/ffmpeg/ffmpeg/compare/master...Kwiboo:drmprime-format) and [mailing list](https://www.mail-archive.com/ffmpeg-devel@ffmpeg.org/msg83572.html), reminder to myself: I really need to send a v2 series to ffmpeg-devel.
As soon as the ffmpeg patch has been merged upstream this could easily be changed to a ffmpeg version check.

This compile test makes it possible to use drm prime video codec for vaapi decoding using my LE [master-rpi-4.19](https://github.com/Kwiboo/LibreELEC.tv/commits/master-rpi-4.19) and Kodi [drmprime-vaapi](https://github.com/Kwiboo/xbmc/commits/drmprime-vaapi) branch.